src/pattern-editor/components/PatternEditorSettingsPanel.tsx

```diff
@@ -48,6 +48,8 @@ export function PatternEditorSettingsPanel() {
 					options={ [ { label: __( 'Select a block' ), value: '' }, ...options ] }
 					onChange={ updateBlockTypes }
 					value={ blockType }
+					__next40pxDefaultSize
```

Comment:
I've been fixing all the deprecation warnings in the console, so this is part of that